cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,46 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+import { VerifyValidator, ModeratePost } from '@joystream/metadata-protobuf'
+import { metadataToString } from '../../helpers/serialization'
+
+export default class VerifyValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static flags = {
+    memberId: flags.string({
+      required: true,
+      description: 'Membership ID of the validator to verify',
+    }),
+    isVerified: flags.boolean({
+      required: true,
+      description: 'Verification state of the validator',
+    }),
+
+    ...WorkingGroupsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { memberId, isVerified } = this.parse(VerifyValidatorAccountCommand).flags
+
+    const meta = new ModeratePost({
+      VerifyValidator: new VerifyValidator({
+        memberId,
+        isVerified,
+      }),
+    })
```

Comment:
The build is failing because the message construction is invalid. Also, why are you wrapping the `VerifyValidator` messaging in `ModeratePost`, please look at the protobuf message structures to encode the message properly https://github.com/chrlschwb/joystream/blob/2bfefe793774cff2fd3699db1df06da9fc300846/metadata-protobuf/proto/WorkingGroups.proto#L66

Reply:
please check again